### DISCUSSION TITLE
Optional Selected Mods Leaderboards

### DISCUSSION THREAD

**Discussion by nobbele** [ppy/osu#13020] [Ideas]
Being able to select optional mods in the leaderboard like if you select HR mod and HD as optional it should show both HR and HDHR plays. This is only really useful for HD so it would be fine if only a "Include Hidden" button was there

**Comment by peppy** [comment]
Is this already present on the web implementation?

**Comment by nobbele** [comment]
i don't know, osu web is ridiculous. [look at this](https://imgur.com/a/Xes3E91), selecting HD gives HDDT but selecting HDDT gives NC?

**Comment by Aergwyn** [comment]
I get `Non-Image content-type returned` for that link.

**Comment by nobbele** [comment]
wtf github, fixed it

**Comment by peppy** [comment]
I believe that means web is using inclusive or, not exclusive and. There's arguments for either way, or having both (but the UI will need to be well though as to not be over-complicated for the average user).

I'd recommend making an issue over at osu-web as if this is going to get fixed, it will happen there first.
